RecoHI/HiEvtPlaneAlgos/src/EvtPlaneProducer.cc

```diff
@@ -66,28 +91,42 @@ class EvtPlaneProducer : public edm::EDProducer {
       etamax2=etamaxval2;
       sumsin=0;
       sumcos=0;
+      sumsinNoWgt=0;
```

Comment:
this should be made a edm::stream::EDProducer after
- TFileService is removed
- ESWatcher is used